templates/connector/package.json

```diff
@@ -1,21 +1,8 @@
 {
-  "name": "{{MANIFEST_NAME}}",
   "dsccConnector": {
     "production": "{{PRODUCTION_DEPLOYMENT_ID}}",
     "latest": "{{LATEST_DEPLOYMENT_ID}}"
   },
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "MIT",
```

Comment:
This was removed because this is a template file, correct? One which will be copied into the user project directory?

Reply:
Correct. That probably shouldn't have ever been there. Better to let the user decide which license they want.